extension/flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisExecutionOptions.java

```diff
@@ -105,6 +111,25 @@ public Builder setStreamLoadProp(Properties streamLoadProp) {
             return this;
         }
 
+        public Builder setJsonFormat() {
+            this.streamLoadProp.setProperty("format", "json");
+            this.streamLoadProp.setProperty("strip_outer_array", "true");
```

Comment:
the `strip_outer_array` is not always true. This should leave to user to decide.

And you don' t need to open another PR to modify this code. You can just push a new commit to your own branch
`feature_add_batch_delete_support`, and Github will automatically sync that commit to this PR.